gpage.h

```diff
@@ -58,17 +58,17 @@ namespace gcpp {
 
 		//	Copy and move are disabled by const unique_ptr member, but let's be explicit
 		//
-		gpage(gpage&) = delete;
-		void operator=(gpage&) = delete;
+		gpage(const gpage&) = delete;
+		gpage& operator=(const gpage&) = delete;
```

Comment:
This is purely stylistic, and I have to agree that's a valid choice. I'll revert this.